docs/docs/getting-started/credentials-tutorial.mdx

```diff
@@ -53,7 +53,140 @@ Integrating the Credentials Provider is as simple as initializing it in the Auth
 
   </TabItem>
   <TabItem value="sveltekit" label="SvelteKit">
-    TODO SvelteKit
+
+  ### Prerequisites
+
+  This tutorial assumes you have a SvelteKit application set up. If you don't, you can follow the [SvelteKit tutorial](https://kit.svelte.dev/docs/creating-a-project) to get started.  
+
+  ### Installing Auth.js
+
+  ```bash npm2yarn
+  npm install @auth/core @auth/sveltekit
+  ```
+
+  ### Adding environment variables
+
+  In your project root, create a `.env.local` file and add the `AUTH_SECRET` environment variable:
+
+  ```title=".env.local"
+  AUTH_SECRET="This is an example"
+  ```
+
+  `AUTH_SECRET` is a random string used by the library to encrypt tokens and email verification hashes, and **it's mandatory to keep things secure**! 🔥 🔐 . You can use:
+
+  ```
+  openssl rand -base64 32
+  ```
+
+  or https://generate-secret.vercel.app/32 to generate a random value for it.
+
+  ### Create server hook
+
+  Create the following [Server hook](https://kit.svelte.dev/docs/hooks) file. This route contains the necessary configuration for Auth.js, as well as the dynamic route handler:
+
+  ```ts title="src/hooks.server.ts"
+  import { SvelteKitAuth } from "@auth/sveltekit"
+  import Credentials from '@auth/core/providers/credentials'
+
+  export const handle = SvelteKitAuth({
+    providers: [
+      Credentials({
```

Comment:
This would be for sign-in right? How would the user sign-up?
For example, I'm using `drizzle adapter` and here's the schema for the user:
```ts
export const users = pgTable('user', {
	id: text('id').notNull().primaryKey(),
	name: text('name'),
	email: text('email').notNull(),
	emailVerified: timestamp('emailVerified', { mode: 'date' }),
	image: text('image')
});
```
Should I add new fields here for username and password?
It would be nice to know how to sign up on this doc as well.